docs/guides/cloud-setup.mdx

```diff
@@ -26,11 +26,11 @@ Your user account is associated with one or more [instances](./instances) that g
 ## Set up to use IBM Cloud
 
 1. Before setting up with IBM Cloud, ensure you are working in an active Python environment with the [Qiskit SDK and Qiskit Runtime installed](/docs/guides/install-qiskit#local).
-1. If you do not already have one, set up an [IBM Cloud account.](https://cloud.ibm.com/registration?cm_sp=quantum&utm_content=quantum-next&target=https%3A%2F%2Fquantum.cloud.ibm.com&error_uri=)
+1. If you do not already have one, set up an [IBM Cloud account](https://quantum.cloud.ibm.com/registration).
    <Admonition type="note">
    You can have multiple IBM Cloud accounts.  You can access any of your IBM Cloud accounts at any time from the account switcher in the top header of the [IBM Quantum Platform interface](https://quantum.cloud.ibm.com).
    </Admonition>
-1. Create an instance, if necessary. Go to the [IBM Quantum Platform dashboard.](https://quantum.cloud.ibm.com) If you have one or more instances shown, continue to the next step. Otherwise, follow these steps to create an instance.
+1. Create an instance, if necessary. Go to the [IBM Quantum Platform dashboard](https://quantum.cloud.ibm.com). If you have one or more instances shown, continue to the next step. Otherwise, follow these steps to create an instance.
```

Comment:
```suggestion
1. Create an instance, if necessary. Go to the [IBM Quantum Platform dashboard.](https://quantum.cloud.ibm.com) If you have one or more instances shown, continue to the next step. Otherwise, follow these steps to create an instance.
```